controlplane/package.json

```diff
@@ -88,6 +92,7 @@
     "eslint": "^8.52.0",
     "eslint-config-unjs": "^0.2.1",
     "eslint-plugin-require-extensions": "^0.1.3",
+    "pg": "^8.11.3",
```

Comment:
This is not needed. Please use drizzle or postgres.js driver.

```ts
  const queryConnection = postgres(databaseConnectionUrl);
  const db = drizzle(queryConnection, { schema: { ...schema } });
```

Reply:
Ah i forgot to remove it, it is required to run drizzle studio